fastlane_core/spec/helper_spec.rb

```diff
@@ -178,7 +180,13 @@
       context "#itms_path" do
         it "default", requires_xcode: true do
           stub_const('ENV', { 'FASTLANE_ITUNES_TRANSPORTER_PATH' => nil })
-          expect(FastlaneCore::Helper.itms_path).to match(/itms/)
+
+          if FastlaneCore::Helper.xcode_at_least?("14")
+            expect(FastlaneCore::UI).to receive(:user_error!).with(/Could not find transporter/)
+            FastlaneCore::Helper.itms_path
```

Comment:
What means this line?

Reply:
For this https://github.com/fastlane/fastlane/pull/20631#discussion_r967653482 (I think, because i did not commit it)

Reply:
This is testing the expected behaviour when fetching itms_path in Xcode14
please see: https://github.com/fastlane/fastlane/blob/ff41f3556f1baf8e700477e50f65be728780aa94/fastlane_core/lib/fastlane_core/helper.rb#L244

<img width="575" alt="Screenshot 2022-09-11 at 09 00 19" src="https://user-images.githubusercontent.com/5364500/189516258-6bf0fb46-eafc-4a30-9c13-118e08387ba0.png">

Reply:
Let's use `not_to raise_error` to check `itms_path` to work well.

```ruby
expect { FastlaneCore::Helper.itms_path }.not_to raise_error
```

It helps be clear why we called this property here.